app/src/main/java/a3/com/convo/adapters/AdditionalLikeAdapter.java

```diff
@@ -12,9 +12,16 @@
 
 import a3.com.convo.R;
 
+/** The AdditionalLikeAdapter takes care of the function of adding likes to your profile that were
+ * not automatically added through the pageLikes data from Facebook upon login. It allows the user
+ * to enter a subject and insert it to a list of other likes, as well as delete them.
+ *
+ * In the future: We will have this search existing pages on Facebook to add instead of text subjects.
+ */
+
 public class AdditionalLikeAdapter extends RecyclerView.Adapter<AdditionalLikeAdapter.ViewHolder> {
     private List <String> mLikes;
-    Context context;
+    private Context context;
```

Comment:
We dont need to save context as a member variable as long as we have access to the view